packages/gantt/frontend/sirius-components-gantt/src/representation/Gantt.tsx

```diff
@@ -38,6 +42,12 @@ const useGanttStyle = makeStyles((theme) => ({
   ganttContainer: {
     backgroundColor: theme.palette.background.default,
   },
+  collapseToggleArrow: {
+    cursor: 'pointer',
+  },
+  noChildren: {
+    paddingLeft: '20px',
```

Comment:
rather than arbitrary values, we always try to use `theme.spacing`

Reply:
Yes but for this case, I depend on this arbitrary value used in gantt-task-react